src/components/DisplayDate/DisplayDate.tsx

```diff
@@ -3,10 +3,18 @@ import moment from 'moment';
 export type DisplayDateProps = {
   isoDate: string;
   format?: string;
+  utc?: boolean;
 };
 
-function DisplayDate({ isoDate, format = 'MMMM Do YYYY, HH:mm' }: DisplayDateProps) {
-  return <>{moment(isoDate).format(format)}</>;
+function DisplayDate({
+  isoDate,
+  format = 'MMMM Do YYYY, HH:mm',
+  utc = false,
+}: DisplayDateProps) {
+  if (utc) {
+    return moment.utc(isoDate).format(format);
+  }
+  return moment(isoDate).format(format);
```

Comment:
Would `return moment.utc(isoDate).format(format);` just return the same thing without the `if` statement?

Reply:
No, the results are different. 
- `moment('2024-01-30T08:11:00+00:00').format(...)` gives in local time zone, so it will be `January 30th 2024, 09:11 (UTC +01:00)`
- `moment.utc('2024-01-30T08:11:00+00:00').format(...)` is displaying in UTC timezone `January 30th 2024, 08:11`

But to improve the readability we could do something like this:
```suggestion
  const momentParser = utc ? moment.utc : moment;
  return momentParser(isoDate).format(format);
```

I am happy to do that 😁

Reply:
No need to change anything, it is fine as it is. You've answered my question, thanks :+1: